CHANGELOG.md

```diff
@@ -6,19 +6,19 @@
 
 - Export writeFinalizeWithdrawTransaction and fix getProveWithdrawalTransactionArgs
 
-## 0.0.1-alpha.5
```

Comment:
@Sabnock01 I think read finalizedWithdrawals first shows up here https://www.npmjs.com/package/op-viem/v/0.0.1-alpha.5?activeTab=code

I think possibly .2 was a no-op and the latest should have been .6, matching https://www.npmjs.com/package/op-viem/v/0.0.1-alpha.5?activeTab=versions

Reply:
Got it. So the only issue was the most recent was incorrectly labeled `.5` instead of `.6`.